CharacterCreation/01_02_Species_ECSC_Security_Drone.txt

```diff
@@ -163,14 +163,22 @@ Opalite scale undulations. Unfortunately, this augmentation does take the slot
 that Drone IR filters would otherwise inhabit, requiring the use of external 
 goggles for such night vision.
 
+    Characters with D-Filters are immune to the blindness of flashbangs. 
+Opalites that use Scale abilities don't get the bonus for those abilities toward
+their charisma checks against them.
+
 Flame Retardant:
     Being able to quickly radiate and dump heat quickly makes many of the Cinder
-Series Drones partially resistant to heat damage. They only take 50% of damage
-from flame or lava, and plasma burn only lasts 1 turn per stack instead of 2. 
-However, these systems are not well-calibrated for extreme cold. Long exposure
+Series Drones partially resistant to heat damage. They take half damage from 
+high-temperature Thermal damage sources, plasma burn only lasts 1 turn per stack 
+instead of 2, and they are immune to status effects applied by high-temperature 
+environmental conditions. 
+
+    However, these systems are not well-calibrated for extreme cold. Long exposure
 can lead to material embrittlement and coolant viscosity fluctuation. A Flame
-Retardant Drone takes 25% more damage from cryo weaponry, environmental 
-hazards on ice planets or similar effects. 
+Retardant Drone takes 25% more damage from low-temperature Thermal damage sources
+and numerical effects from negative status effects applied by low-temperature 
+environmental conditions are increased by 50%. 
```

Comment:
I don't love how this became twice as wordy. But at this point, I'm not sure it can be helped.

Reply:
I think we are going to handle the Themal/Heat/Cold issue in a later update.

Reply:
That makes a lot of sense. I think so too.